doc/netplan.md

```diff
@@ -272,6 +272,12 @@ Virtual devices
     Stateless Address Autoconfiguration (only supported with `NetworkManager`
     backend). Possible values are ``eui64`` or ``stable-privacy``.
 
+``ipv6-address-token-id`` (scalar) – since **0.100**
+
+:   Define an IPv6 address token for creating a static interface identifier for
+    IPv6 Stateless Address Autoconfiguration. This implies a slightly modified
+    ``eui64`` mode and is mutually exclusive with ``ipv6-address-generation``.
+
```

Comment:
I agree with the other PR comments: "token id" is redundant.

I also think that "token" itself is unclear; and at least in networkd, token is used to mean something other than just the id.

https://tools.ietf.org/html/rfc4862 mentions in Appendix B that they /used to/ use the term "interface token" but now say "interface identifier".

At https://developer.gnome.org/NetworkManager/stable/settings-ipv6.html, NetworkManager references draft-chown-6man-tokenised-ipv6-identifiers-02 for its definition of "token". But this also refers to "tokenized interface identifier" - so "interface identifier" is used consistently.

Therefore I recommend "ipv6-interface-identifier" as the key for this.

Reply:
The term used in iproute2, NetworkManager, systemd, etc. is "token." The token is **not** an interface identifier; labeling it as such is extremely misleading. The accurate key for this is, as stated, `ipv6-address-token`.

Reply:
I agree it's important that this be discoverable for people who are looking for "token", given that this term is used throughout iproute2, NetworkManager, and networkd as you mention (though, not with a consistent meaning).

Given that the term 'token' is deprecated in the RFCs, however, I don't like `ipv6-token-id` as this propagates the idea of a 'token' as something that new users are expected to understand.

My suggestions are either to use the `ipv6-interface-identifier` and reference 'token' in the documentation; or use `ipv6-address-token`.

Reply:
Thank you very much @cluonbeam and @vorlonofportland for your input!
I'm going with `ipv6-address-token` then, to keep it in line with the related `ipv6-address-generation` setting.